### ISSUE TITLE
JiraUpdateTicket.gs

### ISSUE THREAD

**Issue by paul-lemon** [ljay79/jira-tools#139]
Name would be better as UpdateJiraTicketManager

Single Responsibility - Update a series of JIRA tickets using the same column / field defintions
Use of a class will enable refactoring
1. Reduce length and complexity of method updateJiraIssues (approx 70 lines)
2. Remove need for convoluted error handling of field names identified in https://github.com/ljay79/jira-tools/pull/130 
`if (responseData.errors != null) {
          Object.keys(responseData.errors).forEach(function (fieldid) {
            jiraErrorMessage = jiraErrorMessage + "{Field:" + fieldid + "}: " + responseData.errors[fieldid] + ", ";
          });
        }

`

**Comment by ljay79** [comment]
We wont need to include the string "jira" everywhere :) as the entire code base is all about jira.

`Ticket` vs `Issue`
I notice myself, mixing up the same thing calling it sometimes _Ticket_ and other time _Issue_. 
Lets try (going forward) to stick to Atlassian terminology: **Issue**

So my name suggestion including above statements would be:
**IssueUpdateManager**

**Comment by paul-lemon** [comment]
agreed - I think some naming conventions in a Contributing.MD to avoid future naming mishaps too.

**Comment by ljay79** [comment]
Sure, as soon i have more than 1 convention :)

**Comment by paul-lemon** [comment]
packageRowForUpdate should be private also.

**Comment by paul-lemon** [comment]
This is underway with the re-factoring work I am doing.
